bool manual_mode

int8 vel_motor_left
int8 vel_motor_right

int8[] matrix1
int8[] matrix2
